engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/AnyToTextNode.java

```diff
@@ -53,20 +49,22 @@ private Text consName(AtomConstructor constructor) {
   }
 
   @CompilerDirectives.TruffleBoundary
```

Comment:
That seems spurious after the change?

Reply:
I am used to leave _string concatenation_ behind `@TruffleBoundary`. In this case, it is just a _rope concatenation_ - e.g. it could be performed in PE mode. However: does it make sense? Will it speed something significant up?

Unless there is a benchmark that gets improved, let's leave `@TruffleBoundary` there. The space for compiled code is limited and we should reserve it for important code that has to run fast. String concatenation is unlikely such code.